Marlin/example_configurations/Hephestos/Configuration_adv.h

```diff
@@ -374,7 +374,15 @@ const unsigned int dropsegments=5; //everything with less than this number of st
 
 //The ASCII buffer for receiving from the serial:
 #define MAX_CMD_SIZE 96
-#define BUFSIZE 5
+#define BUFSIZE 4
```

Comment:
I'm not sure why Witbox and Hephestos have a larger number here, but this seems fine to me.